lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -20,13 +23,21 @@ def include_paths
         end
 
         def languages
-          config.fetch("languages", {})
+          config.fetch("languages", ["ruby"])
```

Comment:
This doesn't seem necessary here? The QM branch of CLI will provide this as the default anyway: https://github.com/codeclimate/codeclimate/blob/quality-model/lib/cc/config/json.rb#L40. It doesn't seem necessary for the check-config changes here, so I think we can keep this as-is.